### DISCUSSION TITLE
Documentation on change date format

### DISCUSSION THREAD

**Discussion by luisjuniorj** [strapi/strapi#7740] [Legacy Topics]
Documentation on change date format

**Comment by derrickmehaffy** [comment]
Can you please be more specific? Change date format where?

**Reply by luisjuniorj** [reply]
Change de date format on datepicker.

**Reply by derrickmehaffy** [reply]
Can you provide an example of what you want to see?

**Reply by luisjuniorj** [reply]
On the datepicker to be able to set date format like DD/MM/YYYY or at least YYYY/MM/DD

**Comment by MattieBelt** [comment]
@luisjuniorj, this isn't possible in the admin panel. 
It looks like [the component lib](https://github.com/strapi/buffet/blob/0c83c30e01cf5ff760c3d7cac82c8719ae061552/packages/buffetjs-core/src/components/DatePicker/index.js#L28) could support it, but I haven't tested it.

**Reply by derrickmehaffy** [reply]
@soupette didn't we add something to adjust the format? I was digging around and I couldn't find anything, but I could of swore I remembered seeing something like this.

**Reply by MattieBelt** [reply]
Doesn't look like it, but it would be nice if it was :)
https://github.com/strapi/strapi/blob/767c3e30c00bda1e924ae5d61202791fed61e790/packages/strapi-plugin-content-type-builder/controllers/validation/types.js#L214-L222

**Reply by soupette** [reply]
Hello @luisjuniorj!

In order to change the display format of the date picker you need to override the following [file](https://github.com/strapi/strapi/blob/9461f20da06b4b57461b242815bbee2391720749/packages/strapi-plugin-content-manager/admin/src/components/Inputs/index.js#L267-L297) in your application, don't hesitate to check out our documentation to do so, by adding the `displayFormat="DD/MM/YYYY"` prop to the `<Input />` component.

Regarding the List view you need to look at this [one](https://github.com/strapi/strapi/blob/master/packages/strapi-plugin-content-manager/admin/src/utils/dateFormats.js).

Good luck!
